easybuild/easyconfigs/p/ParaView/ParaView-5.11.1-foss-2022b-CUDA-12.2.0.eb

```diff
@@ -0,0 +1,79 @@
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.11.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://www.paraview.org'
+description = "ParaView is a scientific parallel visualizer."
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+local_download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['https://www.paraview.org/paraview-downloads/%s' % local_download_suffix]
+sources = ["%(name)s-v%(version)s.tar.gz"]
+patches = ['ParaView-5.11.1-remove_glew_init_warning.patch',
+           ('ParaView-5.11.1-fix_thrust_vtk_m.patch', 'VTK/ThirdParty/vtkm/vtkvtkm/vtk-m/')]
+checksums = [
+    {'ParaView-v5.11.1.tar.gz': 'de32f3e576b5f639ffc6903aa9e4cd46ac30c753185edc4366a7f305a6951b16'},
+    {'ParaView-5.11.1-remove_glew_init_warning.patch':
+     'dd86134f3a5b2c1b834224c69665dd31f99ef7d367688fe77dbaada212758710'},
+    {'ParaView-5.11.1-fix_thrust_vtk_m.patch':
+     'cb4a0dacf1fe2ddefa68ea6f6a061e6cb7d21717053a6514679f9ed73a52525f'},
+]
+
+builddependencies = [('CMake', '3.24.3')]
+
+dependencies = [
+    ('CUDA', '12.2.0', '', SYSTEM),
+    ('OptiX', '7.2.0', '', SYSTEM),
+    ('Boost', '1.81.0'),
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+    ('XZ', '5.2.7'),
+    ('HDF5', '1.14.0'),
+    ('netCDF', '4.9.0'),
+    ('libdrm', '2.4.114'),
+    ('Mesa', '22.2.4'),
+    ('Qt5', '5.15.7'),
+    ('zlib', '1.2.12'),
+    ('FFmpeg', '5.1.2'),
+    ('Szip', '2.1.1'),
+]
+
+# Paraview
+configopts = '-DPARAVIEW_INSTALL_DEVELOPMENT_FILES=ON -DPARAVIEW_BUILD_SHARED_LIBS=ON '
+configopts += '-DPARAVIEW_USE_MPI=ON '
+# Enable NVidia IndeX
+configopts += '-DPARAVIEW_USE_CUDA=ON '
+configopts += '-DPARAVIEW_ENABLE_FFMPEG=ON '
+configopts += '-DPARAVIEW_USE_PYTHON=ON '
+# Enable OptiX raytracing
+configopts += '-DPARAVIEW_ENABLE_RAYTRACING=ON '
+configopts += '-DVTK_ENABLE_OSPRAY=OFF '
+configopts += '-DVTK_ENABLE_OPTIX=ON '
+configopts += '-DPARAVIEW_ENABLE_XDMF2=ON '
+configopts += '-DPARAVIEW_ENABLE_XDMF3=ON '
+configopts += '-DPython3_ROOT_DIR=$EBROOTPYTHON '
+configopts += '-DCMAKE_CUDA_ARCHITECTURES=70 72 80 86 87 89 90 90a '
```

Comment:
This does not work, it would need " around the list which should be ";" separated, and the list should not be hardcoded but taken from the cuda_compute_capabilities parameter like this
-DCMAKE_CUDA_ARCHITECTURES="%(cuda_cc_cmake)s"